src/main/java/cz/kominekjan/disenchantment/utils/DisenchantUtils.java

```diff
@@ -50,4 +56,40 @@ public static Boolean canBeEnchanted(Material material) {
     public static Boolean canBeEnchanted(ItemStack item) {
         return Arrays.stream(enchantmentCheck).anyMatch(e -> e.canEnchantItem(item));
     }
+
+    public static HashMap<Enchantment, Integer> fetchEnchantments(ItemStack item) {
+        HashMap<Enchantment, Integer> enchantments;
+
+        if (item.hasItemMeta() && item.getItemMeta() instanceof EnchantmentStorageMeta meta) {
+            enchantments = new HashMap<>(meta.getStoredEnchants());
+        } else {
+            enchantments = new HashMap<>(item.getEnchantments());
+        }
+
+        HashMap<String, IPlugin> activatedPlugins = PluginManager.getActivatedPlugins();
+
+        if (!activatedPlugins.isEmpty()) {
+            for (IPlugin plugin : activatedPlugins.values()) {
+                plugin.fetchComplementaryEnchantment(item, enchantments);
+            }
+        }
```

Comment:
this here for future plugins or....? Rn does not do anything, default method fetchComplementaryEnchantment is not implemented on any of the supported plugins